### PR TITLE
[docs] Add a recipe about TeamCity integration

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
   * [Debugging with Chrome Developer Tools](articles/documentation/recipes/debugging-with-chrome-dev-tools.md)
   * [Debugging with Visual Studio Code](articles/documentation/recipes/debugging-with-visual-studio-code.md)
   * [Finding Code Issues with Flow Type Checker](articles/documentation/recipes/finding-code-issues-with-flow-type-checker.md)
+  * [Integrating TestCafe with TeamCity CI System](articles/documentation/recipes/integrating-testcafe-with-team-city-ci-system.md)
```
Comment (helen-dikareva) [inline docs/README.md]: May be we should add special sections at least for 'Integration with CI Systems'?

Comment (VasilyStrelyaev) [reply docs/README.md]: Yep, I'm definitely going to do this

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**